app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
```

Comment:
`session` not needed.
Please, use `cookies` with `remember_token`

Reply:
Thank you for your helpful code review. We made the first required changes.

Although, we could not really understand why we need to change from sessions to permanent cookies. We were confused about that and don't know how to do it.